app/api/routers/breeze_buddy/knowledge_base/__init__.py

```
@@ -0,0 +1,417 @@
+            current_user.username,
+        )
+
+        return UploadResponse(
+            status="success",
+            gcs_path=gcs_path,
+            size_bytes=len(content),
+            message=f"File uploaded to {gcs_path}. Run /knowledge-base/index to rebuild the index.",
+        )
+    except HTTPException:
+        raise
+    except Exception as exc:
```

Ruff (0.15.11)

[warning] Do not catch blind exception: `Exception`

(BLE001)